.jscsrc

```diff
@@ -0,0 +1 @@
+node_modules/mofo-style/linters/.jscsrc
```

Comment:
This symlink approach doesn't work on Windows; see https://github.com/mozilla/id.webmaker.org/blob/c5ed8d8d59d53fbb620d163491ce3e43c79fbd19/package.json#L13-L14 for a workaround